Makefile

```diff
@@ -258,15 +258,6 @@ test:
 	git submodule update --init --recursive
 	prove -I../test-nginx/lib -I./ -r -s t/
 
-### license-check:    Check Lua source code for Apache License
```

Comment:
We may still reserve this and also use skywalking-e2e, so that people can still validate licenses on their local env.